puppet/zulip_ops/manifests/profile/nagios.pp

```diff
@@ -16,7 +16,10 @@
   $nagios_mail_domain = zulipconf('nagios', 'mail_domain', undef)
   $nagios_mail_host = zulipconf('nagios', 'mail_host', undef)
   $nagios_mail_password = zulipsecret('secrets', 'nagios_mail_password', '')
-  $nagios_camo_check_url = zulipconf('nagios', 'camo_check_url', undef)
+  if zulipconf('nagios', 'camo_check_url', undef) =~ /^https:\/\/([^\/]*)(\/.*)$/ {
+    $nagios_camo_check_host = $1
+    $nagios_camo_check_path = $2
+  }
```

Comment:
Or we could split this into two configuration options, if we remember to update our `zulip.conf` on deploy.